uktrade/data-hub-frontend
# Update address form to include administrative area for US/Canadian companies.

## Description of change

Adds a field to the address form for US and Canadian companies to record the administrative area.

## Test instructions

If you attempt to add a company that is not matched in D&B's database, you should be able to manually enter details for a company.
If that company is based in the US or Canada, you should have an additional State or Province field.

## Screenshots
### Before

![image](https://user-images.githubusercontent.com/20663545/120654363-89633800-c479-11eb-8d7b-463da0f7779e.png)

### After

![image](https://user-images.githubusercontent.com/20663545/120654014-2d001880-c479-11eb-921a-7cbd9eef5c56.png)

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
